### PR TITLE
Add setMultiplier method to anchor

### PR DESCRIPTION
**Pull request by gmlwhdtjd**
- code cleanup

### CODE REVIEW

**Review by oozoofrog — APPROVED** [review]
👍

File: Sources/SwiftLayout/Components/Anchors/Anchors.swift [hunk 1]
```diff
@@ -27,25 +27,7 @@ public final class Anchors: Constraint {
...
-            constraints.append(NSLayoutConstraint(item: from,
-                                                  attribute: attribute,
-                                                  relatedBy: relation,
-                                                  toItem: to,
-                                                  attribute: toAttribute,
-                                                  multiplier: multiplier,
-                                                  constant: constant))
-        }
-        return constraints
+        constraints(item: fromItem, toItem: toItem, viewInfoSet: nil)
```
Comment (oozoofrog) [inline Sources/SwiftLayout/Components/Anchors/Anchors.swift:30]: 👍

**Review by oozoofrog — APPROVED** [review]
👍

File: Sources/SwiftLayout/Components/Anchors/Anchors.swift [hunk 2]
```diff
@@ -27,25 +27,7 @@ public final class Anchors: Constraint {
...
     }
     
     public func constraints(item fromItem: NSObject, toItem: NSObject?, viewInfoSet: ViewInformationSet?) -> [NSLayoutConstraint] {
```
Comment (oozoofrog) [inline Sources/SwiftLayout/Components/Anchors/Anchors.swift:33]: @gmlwhdtjd how about make viewInfoSet has default parameter value to nil? and remove above function
I thought this is  not so frequently using thing not like equeTo etc.

**Review by oozoofrog — APPROVED** [review]
👍

File: Sources/SwiftLayout/Components/Anchors/Anchors.swift (hunk 2 above)
Comment (gmlwhdtjd) [reply Sources/SwiftLayout/Components/Anchors/Anchors.swift:33]: @oozoofrog  great! I will apply that too

Comment (oozoofrog) [reply Sources/SwiftLayout/Components/Anchors/Anchors.swift:33]: @gmlwhdtjd its my last review. merge plz OTL

**Pull request merged**